_includes/js.html

```diff
@@ -15,3 +15,10 @@
 
     <!-- Custom Theme JavaScript -->
     <script src="{{ "/js/agency.js" | prepend: site.baseurl }}"></script>
+
+     <!-- Syntax Highlighter JavaScript -->
+    <link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/styles/default.min.css">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js">
+    </script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/languages/gherkin.min.js">
+    </script>
```

Comment:
You have to enable highlightjs in order to make it work. Add the following after including the JS files from the CDN:

```html
<script>hljs.initHighlightingOnLoad();</script>
```